pkgs/development/libraries/harfbuzz/default.nix

```diff
@@ -1,13 +1,26 @@
-{ lib, stdenv, fetchFromGitHub, pkg-config, glib, freetype, cairo, libintl
-, meson, ninja
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, glib
+, freetype
+, cairo
+, libintl
+, meson
+, ninja
 , gobject-introspection
-, icu, graphite2, harfbuzz # The icu variant uses and propagates the non-icu one.
-, ApplicationServices, CoreText
+, icu
+, graphite2
+, harfbuzz # The icu variant uses and propagates the non-icu one.
+, ApplicationServices
+, CoreText
 , withCoreText ? false
 , withIcu ? false # recommended by upstream as default, but most don't needed and it's big
 , withGraphite2 ? true # it is small and major distros do include it
```

Comment:
```suggestion
, withGraphite2 ? true # it is small and major distros do include it
, withIntrospection ? stdenv.buildPlatform == stdenv.hostPlatform
```

Reply:
Alternately,

you can define `withIntrospection` in a `let` block as not to clutter the inputs with stuff noone will override anyway.